### DISCUSSION TITLE
Howls not playing until anywhere on the page is clicked.

### DISCUSSION THREAD

**Discussion by HeyNeek** [goldfire/howler.js#1724] [Q&A]
Dealing with an issue where I have Howls that are supposed to play whenever a mouseenter event happens, but no audio will play until anywhere on the screen has been pressed. It's almost as if it's queued all of the noises that should've been playing and then released all at once whenever I finally click somewhere on the page. Then the sounds work perfectly normal after that with the mouseenter events.

**Comment by SuperGamer001** [comment]
This is a normal thing for browsers. Sadly you can't just play audio anytime till the user clicks on the page (as you already well know), unless auto-play is enabled for that site. This is more of a security feature. Really all I can think of to solve this issue is to have a button that users have to click before they get to where you are directing them.
